addons/account_peppol/models/res_partner.py

```diff
@@ -93,4 +93,3 @@ def button_account_peppol_check_partner_endpoint(self):
         edi_identification = f'{self.peppol_eas}:{self.peppol_endpoint}'.lower()
         self.account_peppol_validity_last_check = fields.Date.context_today(self)
         self.account_peppol_is_endpoint_valid = self._check_peppol_participant_exists(edi_identification)
-        return True
```

Comment:
It can't return `None` according to XMLRPC standard spec, so return `False` instead